scripts/php-syntax

```diff
@@ -1,4 +1,14 @@
 #!/bin/bash
 
-# The -P10 option specifies the number of parallel processes (In constrainted CPUs will take approx time for 1 available cpu)
-find . -type f -name '*.php' -not -path '*/vendor/*' -print0 | xargs -0 -n1 -P10 php -l 1>/dev/null
+# Checking if the IS_DEBUG_ENABLED flag is put in the CI environment or a file with the same name exists
+IS_DEBUG_ENABLED=$( [ -n "$IS_DEBUG_ENABLED" ] && echo "$IS_DEBUG_ENABLED" || [ -e "IS_DEBUG_ENABLED" ] && cat "IS_DEBUG_ENABLED" || echo "" )
```

Comment:
@syedahmedhaidershah I find the logic here to be confusing, why would we want to have a file for setting this? Just set it via the environment variable and remove the other use case. Especially since you have labeled it as not recommended.

Reply:
Absolutely ✅

Reply:
@syedahmedhaidershah Thanks for the explanation. I still think it would be better to make this a more deliberate action so we don't have to commit random files to the repository.